runWPT.sh

```diff
@@ -17,7 +17,6 @@ npm run build && \
   --webdriver-arg="--headless=true" \
   --log-wptreport wptreport.json \
   --manifest wpt/MANIFEST.json \
-  --metadata wpt-metadata/mapper/headless \
```

Comment:
Please revert the changes in this file before committing.

Reply:
Actually this script is not used in any of the CI actions, and is used for investigation. So I'd remove the metadata in order to have all the failure details.

Reply:
Given https://github.com/GoogleChromeLabs/chromium-bidi/pull/524, you can just use one of

```
HEADLESS=true ./runWPT.sh
HEADLESS=false ./runWPT.sh
```

...from now on.

Reply:
thanks!